src/components/controls3d/style/controls3d.less

```diff
@@ -50,28 +50,85 @@
       display: inline-block;
       width: 32px;
       height: 30px;
-      background-image: data-uri('left_arrow_redx2.png');
-      background-size: 32px 30px;
+      font-size: 28px;
+      color: #ed1c24;
+      text-shadow: 0px 1px 3px #020202;
 
       @media (max-width: @screen-tablet) {
         display: none;
       }
     }
 
-    .ga-dec {
-      .rotateY(180deg);
-
-      &:active {
-        -webkit-transform: rotateY(180deg) scale(1.2);
-        transform: rotateY(180deg) scale(1.2);
-      }
+    .ga-dec:active {
+      -webkit-transform: scale(1.2);
+      transform: scale(1.2);
```

Comment:
If you don't use the rotate you don't need to redefined the scale because the scale is already defined by `ga-btn`